log_config.go

```diff
@@ -22,6 +22,9 @@ type Config struct {
 	// Include go routine ID as 'gid' in logged fields
 	GoRoutineID *bool `json:"go_routine_id,omitempty"`
 
+	// Include caller info (file:line) as 'caller' in logged fields
+	Caller bool `json:"caller,omitempty"`
```

Comment:
seems weird we don't call these IncludeGoRoutineId / WithGoRoutineId or whatever, they're directives not things.